backend-config/backend-config.go

```diff
@@ -156,6 +156,8 @@ func filterProcessorEnabledDestinations(config ConfigT) ConfigT {
 		source.Destinations = destinations
 		modifiedConfig.Sources = append(modifiedConfig.Sources, source)
 	}
+	modifiedConfig.Settings.DataRetention.DisableReportingPII = config.Settings.DataRetention.DisableReportingPII
```

Comment:
can't we just use the whole `Settings` struct instead of selectively picking fields??

Reply:
DataRetention has other properties that will get overridden, thats why didn't do this

Reply:
Not including workspace settings in the `TopicProcessConfig` looks like a bug to me, cc @cisse21, @BonapartePC

Reply:
Yeah true.... this looks like a bug which we had.